tmt/utils.py

```diff
@@ -125,6 +125,10 @@ def __init__(self, parent=None, name=None, workdir=None, context=None):
         elif workdir is not None:
             self._workdir_init(workdir)
 
+        # Workdir tree is used as the cwd for prepare, execute
+        # and finish steps.
+        self.workdir_tree = None
```

Comment:
Why is it `None`?

Reply:
This is gone. Idea was to have this always initialized directly in the "top" parent class, but I changed it this is not needed anymore. The initialization is do only in `Plan` now.